pkg/ui-client/oss/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": true,
+    "removeComments": true,
+    "preserveConstEnums": true,
+    "outFile": "./dist/index.d.ts"
```

Comment:
As I know, `outFile` is used to indicate output files not file definitions. Currently it indicates on '.d.ts' file. Is it intentionally?

Reply:
not intentional, I might take this out completely now that you've flagged it. We don't really do any TS compilation in these tiny repos.

Reply:
config has been removed since no TS compilation is happening in these projects